src/main/java/com/google/cloud/teleport/templates/common/BigQueryConverters.java

```diff
@@ -174,6 +179,93 @@ public void processElement(ProcessContext context) {
     }
   }
 
+  /** Reads data from BigQuery and converts it to Bigtable mutation. */
+  @AutoValue
+  public abstract static class BigQueryToMutation
+      extends PTransform<PBegin, PCollection<Mutation>> {
+
+    abstract ValueProvider<String> query();
```

Comment:
Related to my question about Classic vs. Flex, if this were a Flex Template, you wouldn't need to wrap these in a `ValueProvider`. The `ValueProvider` is a work-around in Classic Templates, since the graph was pre-serialized and static, but Flex Templates construct the graph during template launching, so options can be passed in just like any other Beam pipeline.